tests/resources/Broker.py

```diff
@@ -1648,6 +1711,31 @@ def compare_rrd_average_value(metric, value: float):
             f"It was impossible to get the average value from the file {VAR_ROOT}/lib/centreon/metrics/{metric}.rrd from the last 30 days")
         return True
 
+##
+# @brief Compare the average value for an RRD metric.
+#
+# @param metric The metric id
+# @param key The key to search in the rrd info
+# @param float The value to compare with.
+#
+# @return A boolean.
+
+
+def compare_rrd_average_value_with_grpc(metric, key, value: float):
```

Comment:
We try to change these comments to something like this:
```
def wait_until_file_modified(path: str, date: str, timeout: int = TIMEOUT):
    """! wait until file is modified
    @param path  path of the file
    @param date  minimal of modified time
    @param path  timeout timeout in seconds
    @return True if file has been modified since date
    """
```